pkg/reconciler/knativeserving/common/extensions.go

```diff
@@ -26,6 +28,9 @@ var log = zap.NewExample().Sugar()
 
 type Platforms []func(kubernetes.Interface, *zap.SugaredLogger) (mf.Transformer, error)
```

Comment:
thoughts on making Platforms[] a struct that holds transformations. That will allow for greater extensibility.

Reply:
I think it worth another PR to extend the Platforms structure.